packages/helpers/classes/request.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+class Request {
+  constructor(opts) {
+    opts = opts || {};
+    this.ANY = '*';
```

Comment:
What's the point/value of this?

Reply:
It was used for Request comparison, but since I took that out with lodash, I should take this out, too.